environments/common/inventory/groups

```diff
@@ -165,3 +162,6 @@ extra_packages
 
 [pulp]
 # Add builder to this group to enable automatically syncing of pulp during image build
+
```

Comment:
Can you move this up to the same place as before please, avoids merge noise/conflicts